inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+
+[[documents_in_getting_started]]
+* *Documents:* Here, you may upload your files to be annotated. Make sure that the format selected in the dropdown on the
+right is the same as the file.
+
+NOTE: *Formats:* For details on the different formats {product-name} provides for importing and exporting single documents as well as whole projects, you may check the main documentation, xref:sect_formats[Appendix A: Formats].
+
+[[users_in_getting_started]]
+* *Users*: Here, you may add users to your _project_ and change
+their rights within that project. You can only add users to a project from the dropdown at the left if they exist already in the {product-name} _instance_.
+
+NOTE: *{product-name} Instance vs. Project:* In some cases, we have to distinguish between the {product-name} *instance* we are working in and the *project(s)* it contains. +
+For example, a user may be added to the {product-name} instance but not to a certain project. Or she may have different rights in several projects.
+
+** *Add new users:* In order to find users for a _project_ in the dropdown, you need to add them to your {product-name} _instance_ first. Click on the *administration*- button in the very top right corner and
+select section *Users* on the left. For *user roles* (for an _instance_ of {product-name}) see the <<User Management>> in the main documentation.
+
+** *Giving rights to users:* After selecting a user from the dropdown in the project settings section *Users*, you can check and uncheck the user‘s rights on the right side. User rights count for that _project_ only and are different from user roles which count for the whole {product-name} _instance_. Any combination of rights is possible and the user will always have the sum of all rights given.
+
+
+[[User_rights]]
+
+[cols=",,",]
+|===
+|User Right | Description | Access to Dashborad Sections
+
+|Annotator |- annotate only |
+- Annotation +
+- Knowledge Base +
+
+|Curator |- curate only|
+- Curation +
+- Monitoring +
+- Agreement +
+- Evaluation
+
+|Project Manager |- annotate +
+- curate +
+- create projects +
+- add new documents +
+- add guidelines +
+- manage users +
+- open documents like an other user sees it (read only) |- All pages
+|===
+
+[[layers_and_features_in_getting_started]]
+NOTE: *Layers and Features:* There are different "aspects" or "categories" you might want to annotate. For example, you might want to annotate all the places and persons in a text and link them to a knowledge base entry (see the boy about xref:knowledge_bases_in_getting_started[Knowledge Bases]) to tell _which_ concrete place or person they are. This way of annotation is called _Named Entity_. In another case, you might want to annotate which words are verbs, nouns, adjectives, prepositions and so on (called _Part of Speech_). What we called "aspects", "categories" or "ways to annotate" here, is referred to as *layers* in {product-name} as in many other annotation tools, too. + 
```

Comment:
That's true, that's why there are paragraphs already...
In case you mean blank lines instead of line breaks only: I'd prefer blank lines, but the box does not allow for that. Do you have a solution for that :-?